src/pymachine/utils.py

```diff
@@ -135,16 +145,11 @@ def to_dot(self):
         # lines.append('\tordering=out;')
         # sorting everything to make the process deterministic
         node_lines = []
-        for node, n_data in self.G.nodes(data=True):
+        for node in self.G.nodes():
```

Comment:
@Eszti did you intend to remove this functionality? I added it recently, perhaps your code wasn't up-to-date?